### PR TITLE
Added: Alert/Notification messages sent to Sendgrid channel

### LINKED ISSUE #323: Alert sendgrid is not yet implemented
**Describe the bug**
The alert for sendgrid channel is not implemented yet

**To Reproduce**
1. Go to file `src/components/notification/index.ts`
2. ctrl/cmd+f `sendgrid`
3. there is no calling of `sendSendgrid` function from `src/components/notification/channel/sendgrid.ts`

**Expected behavior**
There should be a case to call `sendSendgrid` function

### PR DESCRIPTION
**Pull request by raosan**
# Monika Pull Request (PR)  
fix #323

## What feature/issue does this PR add  
1. implement sendgrid alert

## How did you implement / how did you fix it  
1. call function `sendSendgrid` in `src/components/notification/index.ts`

## How to test  
1.  run monika and make an incident on target URL
2. get alert notification with detailed info

### CODE REVIEW

**Comment by raosan** [comment]
need @nicnocquee help to test

**Review by gsukanto — APPROVED** [review]

**Review by armedi — APPROVED** [review]

**Review by nicnocquee — CHANGES_REQUESTED** [review]
Seems like sendgrid changes the URL in the email body. (See my DM in Teams).

**Review by sapiderman — APPROVED** [review]

**Review by dennypradipta — APPROVED** [review]

**Comment by raosan** [comment]
> Seems like sendgrid changes the URL in the email body. (See my DM in Teams).

fixed. please re check

File: src/components/notification/channel/sendgrid.ts [hunk 1]
```diff
@@ -40,7 +40,9 @@ export const sendSendgrid = async (
     to: recipients,
     from: sender.email,
     subject,
-    text: body,
+    text: body.includes('https://')
+      ? body.replace(/https/g, '<https>')
+      : body.replace(/http/g, '<http>'),
```
Comment (nicnocquee) [inline src/components/notification/channel/sendgrid.ts:45]: The email looks weird with <https>. It's better to just use html instead:

```
   subject,
    html: body.replace(/\n/g, '<br/>'),
```

**Review by nicnocquee — CHANGES_REQUESTED** [review]

**Comment by nicnocquee** [comment]
Please also update the documentation to include the `sender` key.

**Review by nicnocquee — APPROVED** [review]

**Pull request merged**